back/engines/commercial/report_builder/spec/services/report_builder/queries/projects_spec.rb

```diff
@@ -9,24 +9,30 @@
     before_all do
       # 2020
       past_project = create(:project)
-      create(:phase, project: past_project, start_at: Date.new(2020, 2, 1), end_at: Date.new(2020, 3, 1))
+      create(
+        :phase,
+        project: past_project,
+        start_at: Date.new(2020, 2, 1),
+        end_at: Date.new(2020, 3, 1),
+        with_permissions: true
+      )
 
       # 2021
       @project1 = create(:project)
-      create(:phase, project: @project1, start_at: Date.new(2021, 2, 1), end_at: Date.new(2021, 3, 1))
+      create(:phase, project: @project1, start_at: Date.new(2021, 2, 1), end_at: Date.new(2021, 3, 1), with_permissions: true)
```

Comment:
Not clear why this needed to be added for this PR?

Reply:
Yeah no clue, I thought it was needed... these tests were failing and I added this when debugging them for reasons I don't remember anymore. But I just removed it and the tests still pass so I will just get rid of it